staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -71,39 +71,39 @@ done
 
 if [ "${GENS}" = "all" ] || grep -qw "deepcopy" <<<"${GENS}"; then
   echo "Generating deepcopy funcs"
-  ${GOPATH}/bin/deepcopy-gen --input-dirs $(codegen::join , "${ALL_FQ_APIS[@]}") -O zz_generated.deepcopy --bounding-dirs ${INT_APIS_PKG},${EXT_APIS_PKG} "$@"
+  "${GOPATH}"/bin/deepcopy-gen --input-dirs "$(codegen::join , "${ALL_FQ_APIS[@]}")" -O zz_generated.deepcopy --bounding-dirs "${INT_APIS_PKG}","${EXT_APIS_PKG}" "$@"
 fi
 
 if [ "${GENS}" = "all" ] || grep -qw "defaulter" <<<"${GENS}"; then
   echo "Generating defaulters"
-  ${GOPATH}/bin/defaulter-gen  --input-dirs $(codegen::join , "${EXT_FQ_APIS[@]}") -O zz_generated.defaults "$@"
+  "${GOPATH}"/bin/defaulter-gen  --input-dirs "$(codegen::join , "${EXT_FQ_APIS[@]}")" -O zz_generated.defaults "$@"
 fi
 
 if [ "${GENS}" = "all" ] || grep -qw "conversion" <<<"${GENS}"; then
   echo "Generating conversions"
-  ${GOPATH}/bin/conversion-gen --input-dirs $(codegen::join , "${ALL_FQ_APIS[@]}") -O zz_generated.conversion "$@"
+  "${GOPATH}"/bin/conversion-gen --input-dirs "$(codegen::join , "${ALL_FQ_APIS[@]}")" -O zz_generated.conversion "$@"
 fi
 
 if [ "${GENS}" = "all" ] || grep -qw "client" <<<"${GENS}"; then
   echo "Generating clientset for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/${CLIENTSET_PKG_NAME:-clientset}"
   if [ -n "${INT_APIS_PKG}" ]; then
-    ${GOPATH}/bin/client-gen --clientset-name ${CLIENTSET_NAME_INTERNAL:-internalversion} --input-base "" --input $(codegen::join , $(printf '%s/ ' "${INT_FQ_APIS[@]}")) --output-package ${OUTPUT_PKG}/${CLIENTSET_PKG_NAME:-clientset} "$@"
+    "${GOPATH}"/bin/client-gen --clientset-name "${CLIENTSET_NAME_INTERNAL:-internalversion}" --input-base "" --input "$(codegen::join , $(printf '%s/ ' "${INT_FQ_APIS[@]}"))" --output-package "${OUTPUT_PKG}"/"${CLIENTSET_PKG_NAME:-clientset}" "$@"
```

Comment:
also here `$(printf '%s/ ' "${INT_FQ_APIS[@]}")`, how to pass https://github.com/koalaman/shellcheck/wiki/SC2046 ? Quoting this will failed as
```Error: Failed making a parser: unable to add directory "k8s.io/code-generator/_examples/apiserver/apis/example/ k8s.io/code-generator/_examples/apiserver/apis/example2/ ": unable to import "k8s.io/code-generator/_examples/apiserver/apis/example/ k8s.io/code-generator/_examples/apiserver/apis/example2/ ": cannot find package "k8s.io/code-generator/_examples/apiserver/apis/example/ k8s.io/code-generator/_examples/apiserver/apis/example2/ " in any of:```

Reply:
Tried again, but same error. This part `$(printf '%s/ ' "${INT_FQ_APIS[@]}")` cannot be quoted.
The problem caused by the quotation marks.

Reply:
@BenTheElder  the problem is this part `--input "$(printf '%s/ ' "${INT_FQ_APIS[@]}")"` will cause `--input 'k8s.io/code-generator/_examples/apiserver/apis/example/ k8s.io/code-generator/_examples/apiserver/apis/example2/ '`

Reply:
aha, using a array can fix this issue. PTAL